### ISSUE TITLE
how to install wavfix

### ISSUE THREAD

**Issue by majedmouj** [agfline/wavfix#10]
Hello @agfline, was wondering if you can help me with installing wavfix on my laptop. i'm kindoff a newbie when it comes to this but i really need help fixing some wav files i have

**Comment by agfline** [comment]
Hi, 
You don't have to install it. Just [download your corresponding OS version](https://github.com/agfline/wavfix/releases) and run it through a console/terminal.
Do you need help with running commands in a terminal ? What is your OS ?

**Comment by majedmouj** [comment]
yes please, I do need help, Im kinda new here. Im running Mac OS Ventura 13.1

**Comment by agfline** [comment]
Ok no problem. Is your Mac an Intel or Silicon ?

**Comment by majedmouj** [comment]
intel

**Comment by agfline** [comment]
Ok that's a good thing.

- First, download https://github.com/agfline/wavfix/releases/download/v0.1/wavfix-0.1-mac64
- Open the finder and go to the directory were you have `wavfix-0.1-mac64` downloaded
- Then you can open a terminal from that directory, [here is a how-to](https://support.apple.com/guide/terminal/open-new-terminal-windows-and-tabs-trmlb20c7888/mac#trml23e1e629)
- Once you're in the terminal, you can run wavfix following the [examples section of the README](https://github.com/agfline/wavfix)

**Comment by agfline** [comment]
Sorry I don't have time for a chat, I take some time to help when I can.
Once you've followed the steps above and your terminal is opened, just type `./wavfix-0.1-mac64 /path/to/your/file.wav` and press enter. wavfix will start running and some text will be printed on the screen. If you read `File successfully recovered` in the end, you're done. If you don't, then you can copy/paste the text output here.

**Comment by spbeck78** [comment]
Hello @agfline. I'm also having issues installing. My PC has Windows 11 on a 64 bit OS. I also have Ubuntu downloaded on my pc, and would like to run wavfix through _that_.

I tried installing both linux64 and win64.exe releases. Neither work.

**Comment by agfline** [comment]
Hi @spbeck78 
You don't have to install wavfix, it's ready to use in a terminal.

On windows, start by opening `CMD` or `Windows PowerShell`, then go to your wavfix folder by typing `cd "C:\absolute\path\to\wavfix\"` then hit enter, then type your wavfix command and hit enter.

On Linux, open a terminal, type `cd "/absolute/path/to/wavfix/"` then hit enter, then type your wavfix command and hit enter.

You can find wavfix command examples [here](https://github.com/agfline/wavfix), don't forget to replace in your command the word `wavfix` with either `./wavfix-0.1-win64.exe` or `./wavfix-0.1-linux64` depending on your OS.
